recaf-assembler/src/main/java/me/coley/recaf/assemble/ast/BaseElement.java

```diff
@@ -149,6 +149,15 @@ public Element getChildAt(int line, int column) {
 
 	}
 
+	public Element getChildAt(int position) {
+		for (Element child : getChildren()) {
+			if (child.getStart() <= position && child.getStop() >= position) {
+				return child;
+			}
+		}
```

Comment:
Optimization: Can use binary search or predictive search on `Element`'s positions to skip over most elements.